check_graphqljson.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+import os
+
+def main(args):
+    if not len(args) == 1:
+        usage()
```

Comment:
Just use `arparse` for this sort of thing, you get usage/help for free. See the get_types.py and decomp.py files for examples